app/graphql/types/profile.rb

```diff
@@ -44,7 +53,47 @@ class Types::Profile < Types::BaseObject
     null: true,
     description: 'The message this user has submitted for the Hall of Fame'
 
+  field :location, String,
+    null: true,
+    description: 'Your general location to be displayed on your profile.'
+
+  field :gender, String,
+    null: true,
+    description: 'What you identify as.'
+
+  field :birthday, GraphQL::Types::ISO8601Date,
+    null: true,
+    description: 'When you were born, or something like that...'
+
+  field :time_zone, String,
+    null: true,
+    description: 'The time_zone for this user.'
+
+  field :country, String,
+    null: true,
+    description: 'The country you are in currently.'
```

Comment:
Neither of these are public, these are private and should be on Account not Profile

Reply:
I totally thought these were on profile, it was actually just location...